hikari/impl/entity_factory.py

```diff
@@ -1989,8 +2028,8 @@ def deserialize_partial_message(  # noqa CFQ001 - Function too long
                 referenced_message = None
 
         stickers: undefined.UndefinedOr[typing.Sequence[message_models.Sticker]] = undefined.UNDEFINED
-        if "stickers" in payload:
-            stickers = [self._deserialize_sticker(sticker) for sticker in payload["stickers"]]
+        if "sticker_items" in payload:
```

Comment:
For backwards compatibility you need to also try to parse "stickers" to a sequence of partial stickers if "sticker_items" isn't present since old messages from before the addition of "sticker_items" will only have "stickers" (and not "sticker_items") when fetched over rest